README.md

```diff
@@ -879,6 +879,7 @@ _Related: [Groupware](#groupware)_
 - [Send](https://github.com/timvisee/send) - Simple, private, end to end encrypted temporary file sharing, originally built by Mozilla. ([Clients](https://github.com/timvisee/send#clients)) `MPL-2.0` `Nodejs`
 - [Torrents.csv](https://codeberg.org/heretic/torrents-csv-server) - A self-hostable torrent search engine. `GPL-3.0` `Rust`
 - [Transmission](https://transmissionbt.com/) - Fast, easy, Free Bittorrent client. ([Source Code](https://github.com/transmission/transmission)) `GPL-3.0` `C`
+- [mork](https://github.com/dojje/mork) - Simple command line based file sending utility, written in rust. ([Source Code](https://github.com/dojje/mork)) `GPL-3.0` `Rust`
```

Comment:
Do not add a duplicate `Source code` link if it is the same as the main link :)

Reply:
Oh right sorry, that makes alot of sense. First pull request ever, so not very good at this lmao. I believe I fixed it now.

Reply:
No worries, that's what the reviews are for :)